docs/getting-started/substrate-essentials.md

```diff
@@ -0,0 +1,26 @@
+---
+description: Substrate knowledge base
+---
+
+# Substrate Essentials
+
+### Substrate
+
+[**`Substrate`**](https://substrate.dev/docs/en/) takes a modular approach to blockchain development and defines a rich set of primitives that allows developers to make use of powerful, familiar programming idioms.
```

Comment:
Remove ` from Substrate. It does not look like a link and no one would know to click it